### PR TITLE
Added documentation regarding the sales channel header

### PR DESCRIPTION
**Pull request by pc-alves**
API was changed to include the `X-Sales-Channel` header. This PR adds that information to the documentation.

### CODE REVIEW

**Comment by s-wi** [comment]
:+1:

**Comment by pc-alves** [comment]
👍

**Comment by pc-alves** [comment]
👍

File: source/includes/_checkout-api.md [hunk 1]
```diff
@@ -704,8 +716,9 @@ Worth noticing is the `payment.external_payment` flag. When set to true, clients
 
 Parameter | Type | Description | Required
 --------- | ---- | ----------- | --------
-OAuth token | Header | Customer token used to authenticate the request. The token also helps validate that the cart and addresses being used belong to the user making the request | Yes
+X-Sales-Channel | Header | Used to specify under which sales channel the request belongs to. The cart specified in the body will have to belong to the same sales channel | Yes
```
Comment (Gregsen) [inline source/includes/_checkout-api.md]: Used to specify **to** which sales channel

File: source/includes/_checkout-api.md [hunk 2]
```diff
@@ -780,9 +793,10 @@ The response is the same object than in checkout creation, so the same considera
 
 Parameter | Type | Description | Required
 --------- | ---- | ----------- | --------
-checkout_id | Path | ID of the checkout | Yes
-OAuth token | Header | Customer token used to authenticate the request. The token also helps validate that the cart and addresses being used belong to the user making the request | Yes
+X-Sales-Channel | Header | Used to specify under which sales channel the request belongs to. The cart originally used to create the checkout will have to belong to the same sales channel | Yes
```
Comment (Gregsen) [inline source/includes/_checkout-api.md]: Used to specify **to** which sales channel

File: source/includes/_checkout-api.md [hunk 3]
```diff
@@ -854,9 +868,10 @@ All operations done on the checkout object can (and should) be done in the check
 
 Parameter | Type | Description | Required
 --------- | ---- | ----------- | --------
-checkout_id | Path | ID of the checkout | Yes
+X-Sales-Channel | Header | Used to specify under which sales channel the request belongs to. The cart originally used to create the checkout will have to belong to the same sales channel | Yes
```
Comment (Gregsen) [inline source/includes/_checkout-api.md]: same her (to instead of under)

File: source/includes/_checkout-api.md [hunk 4]
```diff
@@ -928,8 +943,9 @@ After order creation there might be a final step for payment. Clients of the API
 
 Parameter | Type | Description | Required
 --------- | ---- | ----------- | --------
-OAuth token | Header | Customer token used to authenticate the request. The token also helps validate that the checkout provided to create the order belongs to the user making the request | Yes
+X-Sales-Channel | Header | Used to specify under which sales channel the request belongs to. The checkout passed in the body will have to belong to the same sales channel | Yes
```
Comment (Gregsen) [inline source/includes/_checkout-api.md]: to

File: source/includes/_checkout-api.md [hunk 5]
```diff
@@ -982,10 +999,11 @@ Orders returned are limited to those that were made by the client issuing the re
 
 Parameter | Type | Description | Required
 --------- | ---- | ----------- | --------
+X-Sales-Channel | Header | Used to specify under which sales channel the request belongs to. Only orders created under the same sales channel will be returned | Yes
```
Comment (Gregsen) [inline source/includes/_checkout-api.md]: to

File: source/includes/_checkout-api.md [hunk 6]
```diff
@@ -1046,8 +1065,9 @@ The last two endpoints are read only operations, which means that operations lik
 
 Parameter | Type | Description | Required
 --------- | ---- | ----------- | --------
-order_number | Path | The order number | Yes
+X-Sales-Channel | Header | Used to specify under which sales channel the request belongs to. Only if the order was created under the same sales channel will it be returned | Yes
```
Comment (Gregsen) [inline source/includes/_checkout-api.md]: to

**Comment by Gregsen** [comment]
👍

**Comment by pc-alves** [comment]
👍

**Comment by pc-alves** [comment]
👍

**Comment by hani-ibrahim** [comment]
👍

**Comment by pc-alves** [comment]
👍

**Pull request merged**